secure_tunneling/include/aws/iotsecuretunneling/SecureTunnel.h

```diff
@@ -1130,6 +1146,7 @@ namespace Aws
                 aws_secure_tunneling_local_proxy_mode localProxyMode,
                 const std::string &endpointHost,
 
+                Crt::Io::TlsConnectionOptions *tslOptions,
```

Comment:
Trivial: `tlsoOptions`

Reply:
Good call. Will add it to the documentation